metaflow/plugins/aip/aip.py

```diff
@@ -814,7 +817,7 @@ def _set_container_volume(
                 (resource_op, volume) = self._create_volume(
                     step_name=aip_component.step_name,
                     size=resource_requirements["volume"],
-                    workflow_uid=workflow_uid,
+                    workflow_uid=ARGO_WORKFLOW_UID,
```

Comment:
~~So the `ARGO_WORKFLOW_UID` is a "template" string whereas the `workflow_uid` was the actual UID value previously I presume?~~

Scratch that I see below when this method is called it's identical, just reusing the `const` :)